src/AppBundle/Resources/views/Report/Formatted/Accounts/_final_summary.html.twig

```diff
@@ -34,6 +34,11 @@
                         <td class="grey numeric">&pound;{{ report.getExpensesTotal | money_format}}</td>
                     {% endif %}
                 </tr>
+            {% else %}
+                <tr>
+                    <td>Deputy costs:</td>
+                    <td class="grey numeric">&pound;{{ report.profDeputyTotalCosts | money_format }}</td>
+                </tr>                
```

Comment:
Does this address an issue that has been raised on the Deputy Costs section?

Reply:
I believe so